website/docs/d/is_vpn_gateway.html.markdown

```diff
@@ -89,4 +89,14 @@ In addition to all argument references listed, you can access the following attr
 	- `href` - (String) The URL for this subnet.
 	- `id` - (String) The unique identifier for this subnet.
 	- `name` - (String) The user-defined name for this subnet.
+- `vpc` - (String) 	The VPC this VPN server resides in.
```

Comment:
long space b/w (String) and The VPC.....(in all docs)

Reply:
removed the unwanted spaces.